vocode/streaming/output_device/websocket_output_device.py

```diff
@@ -67,6 +60,6 @@ def consume_transcript(self, event: TranscriptEvent):
     def terminate(self):
         self.process_task.cancel()
 
-    def clear(self):
+    async def clear(self):
```

Comment:
without this change, the only the first ~1 second of audio will play for each response. no clue why